### PR TITLE
Standalone binary build with PyInstaller

### PR DESCRIPTION
**Pull request by miurahr**
- Call PyInstaller directly
- Move build_standalone.py into tools folder
- Remove duplicated configurations from setup.cfg

Signed-off-by: Hiroshi Miura <miurahr@linux.com>

### CODE REVIEW

**Comment by miurahr** [comment]
I made standalone_build script simpler.

@nikitalita could you check a new standalone binary work for you?

**Comment by nikitalita** [comment]
It works, but only after I install the venv manually. I'm kind of surprised that the CI works with bash though; the default Windows runner uses Git Bash, and I've found that using the default Windows python with it will sometimes cause it to behave very oddly. I'd recommend setting up a powershell script for the venv setup steps, as powershell works on all platforms.

**Comment by miurahr** [comment]
@nikitalita Even build on powershell and remove venv after build, it passed.
Could you share reproduce conditions or logs?

**Comment by miurahr** [comment]
@nikitalita  You can download artifact which is built on CI https://github.com/miurahr/aqtinstall/actions/runs/3349589622#artifacts

**Comment by miurahr** [comment]
I've squashed commits.

File: tools/launch_aqt.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!python.exe
+import aqt
+import re
+import sys
+
+if __name__ == '__main__':
+    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
```
Comment (miurahr) [inline tools/launch_aqt.py:7]: Is it better to be just `sys.argv[0] = 'aqt'` ?

File: tools/build_standalone.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        adddata_arg = "{src:s};aqt"
+    else:
+        adddata_arg = "{src:s}:aqt"
+    for data in ["aqt/logging.ini", "aqt/settings.ini", "aqt/combinations.json"]:
+        args.append('--add-data')
+        args.append(adddata_arg.format(src=data))
+    args.append(os.path.join(tools_dir, "launch_aqt.py"))
+
+    # launch PyInstaller
+    PyInstaller.__main__.run(args)
```
Comment (miurahr) [inline tools/build_standalone.py:33]: Here is a way to call PyInstaller from python code.
https://pyinstaller.org/en/stable/usage.html#running-pyinstaller-from-python-code

It is better than `gravitybee` that call `pyinstaller` using `subprocess`.
Now the process is speedy and safe.

**Comment by nikitalita** [comment]
> @nikitalita Even when build on powershell and remove venv after built, it passed on CI. Could you share reproduce conditions or logs?

It works fine. I'm just saying that it MIGHT fail in the future if we keep using `bash`.

File: tools/launch_aqt.py (hunk 1 above)
Comment (miurahr) [reply tools/launch_aqt.py:7]: It is ok for now.

**Comment by miurahr** [comment]
Thank you for feedback. I'd like to merge.

**Pull request closed without merging**